packages/fxa-auth-server/lib/routes/utils/security-event.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { AccountEventsManager } = require  ('../../account-events');
+const { Container } = require('typedi');
+
+export function recordSecurityEvent(name, opts) {
```

Comment:
I can see why you added this logic to a utility function, keeps things nicely separated. However, it does add one more layer to recording the event. Either way we should probably standarized on approach but that can done in another PR.